### ISSUE TITLE
What is voxel file used for during ONet training?

### ISSUE THREAD

**Issue by jnyjxn** [autonomousvision/occupancy_networks#42]
Hello,

Thank you for this great work and clean codebase!

I am trying to relate the training script of your work to the CVPR paper and I want to better understand the use of the voxel data (e.g. "model.binvox"). Is it to decide which coarse region should be sampled from?

By extension, is the performance of ONet highly dependent on the use of this voxel data? For example, does the performance improve if the resolution of the grid is increase from 32x32x32 to 64x64x64?

**Comment by AlexsaseXie** [comment]
Voxel data is only used for evaluation. Nearly all of the deep learning network approaches to do the task of 3D reconstruction use voxel iou as a metric but not a component of the final loss function because iou loss cannot calculate gradients. Increase the relosution of the voxel cannot actually affect the reconstructed result of ONet(for voxel don't participate training) but may make ONet look better than other approach(other approaches usually cannot handle high resolution, or suffer from huge memory space cost). This is one of ONet's superior.
